billy-gin/src/main/java/com/premiumminds/billy/gin/services/ExportService.java

```diff
@@ -31,7 +36,18 @@
 	public <T extends ExportServiceRequest> File exportToFile(T request)
 		throws ExportServiceException;
 
+	@Deprecated
```

Comment:
If you are breaking the functionality it is preferable to also break the API otherwise users might think their code is just working as expected.

Please remove addHandler method.